MerlinAutoUpdate.sh

```diff
@@ -128,26 +134,40 @@ Update_Custom_Settings() {
     esac
 }
 
+##----------------------------------------##
+## Modified by Martinski W. [2023-Oct-05] ##
+##----------------------------------------##
 get_current_firmware() {
-    local current_version="$(nvram get buildno)_$(nvram get extendno)"
+    local current_version="$(nvram get buildno).$(nvram get extendno)"
```

Comment:
Will need to review implications more with further audit of dependencies down the script:

Version Format: If other parts of the script or system expect the version to be in the format of buildno_extendno (e.g., 1234_5678), the modified version will break that expectation. Conversely, if the system or a new standard expects the version to be in the format of buildno.extendno (e.g., 1234.5678), the modification is correct.